buildscripts/incremental/setup_conda_environment.cmd

```diff
@@ -14,9 +14,7 @@ cmd /C conda info
 set CONDA_INSTALL=cmd /C conda install -q -y
 set PIP_INSTALL=pip install -q
 
-@rem Use conda-forge for NumPy 1.24 - at the time of writing it is not available
-@rem on the defaults channel.
-if %NUMPY%==1.24 (set NUMPY_CHANNEL_PKG="conda-forge::numpy") else (set NUMPY_CHANNEL_PKG="numpy")
+set NUMPY_CHANNEL_PKG="numpy"
```

Comment:
I can't make a suggestion on line L28/R26, but that line should change to:

```
conda create -n %CONDA_ENV% -q -y python=%PYTHON% numpy=%NUMPY% cffi pip scipy jinja2 ipython gitpython pyyaml
```

and this line be deleted.